teradata/CHANGELOG.md

```diff
@@ -1,2 +1,5 @@
 # CHANGELOG - Teradata
 
+## 1.0.0 / 2022-05-13
+
+* [Added] Add teradata integration. See [#11701](https://github.com/DataDog/integrations-core/pull/11701).
```

Comment:
Smallest of nits on the capatilization of Teradata, unless the official name is without caps.

Reply:
Updated, thanks!